airbyte-integrations/connectors/source-shopify/source_shopify/streams/base_streams.py

```diff
@@ -642,7 +642,7 @@ def __init__(self, config: Dict) -> None:
         # define BULK Manager instance
         self.job_manager: ShopifyBulkManager = ShopifyBulkManager(
             session=self._session,
-            base_url=f"{self.url_base}/{self.path()}",
```

Comment:
Why has this changed? It does not seem related to the retrying part

Reply:
It was like that:

before: `http://domain.my_shopify.com//graphql.json` - introducing the extra `/` in the path. It seems like the `self.path()` includes this once joined with the `base_url`. No need for that.

After: `http://domain.my_shopify.com/graphql.json`